docs/Upgrade-Guide/upgrade_to_6.md

```diff
@@ -0,0 +1,81 @@
+## Upgrade procedure to Gluster 6, from Gluster 5.x, 4.1.x, and 3.12.x
+
+We recommend you to read [Release notes of v6.0](https://github.com/gluster/glusterfs/blob/v6.0/doc/release-notes/6.0.md) to be aware of the features, and fixes in the release before the upgrade.
+
+> **NOTE:** Upgrade procedure remains the same as with 4.1.x release
+
+Refer, to the [Upgrading to 4.1](./upgrade_to_4.1.md) guide and follow
+documented instructions, replacing 6 when you encounter 4.1 in the guide as the
+version reference.
+
+## Deprecated translator upgrade guide
+
+With glusterfs-6.0 release, we are planning to deprecate few of the
+features (or volume types) which existed before.
+
+If you have any one of the below volume types, please go through
+instructions below, so you can preserve sanity in your glusterfs
+release.
+
+### Stripe volume
```

Comment:
I think we need some rewording here as and like so in other cases as well:

Stripe xlator, provided the ability to stripe data across bricks. This functionality was used to create and support files larger than a single brick and also to provide better disk utilization across large file IO, by spread the IO blocks across bricks and hence disks.

This functionality is now provided by the shard xlator (any links here wold help, to configure and use shard).

There is no in place upgrade feasible for volumes using the stripe feature, and users are encouraged to migrate their data from existing stripe based volumes to sharded volumes.

Reply:
Done